examples/neural_prophet/post_execution.ipynb

```diff
@@ -0,0 +1,149 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "013a8a8d",
+   "metadata": {},
+   "source": [
+    "# Exploring the pipeline\n",
+    "\n",
+    "Here, we use a simple post-execution flow to fetch the a trained pipeline and inspect the results. To run this notebook, please make sure you have run the pipeline defined in `run.py`.\n",
+    "\n",
+    "```python\n",
+    "zenml init\n",
+    "python run.py\n",
+    "```\n",
+    "\n",
+    "And then continue:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64b9c241",
+   "metadata": {},
+   "source": [
+    "## Fetch the results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a0dc8382",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from zenml.repository import Repository"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3ffbc50a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Fetch the pipeline\n",
+    "repo = Repository()\n",
+    "pipe = repo.get_pipelines()[-1]"
```

Comment:
i would advise against using get_pipelines()[-1] for this sort of stuff - if the user does not do a zenml init it is possible for the pipelines to not be chronologically ordered (I've somehow managed that on my machine) - in that case you might be getting a completely different pipeline. This can also happen if you run this example - then some other pipeline and then you come back to this example.

Using get_pipeline(pipeline_name=...) is more reliable